.eslintrc.js

```diff
@@ -436,5 +436,14 @@ module.exports = {
 				'react/react-in-jsx-scope': 'error',
 			},
 		},
+		{
+			files: [
+				'packages/interactivity-router/src/index.js',
```

Comment:
Maybe we should have another group for Interactivity API, which uses Preact behind the scenes. It should be:

- `packages/interactivity-router/src/*`
- `packages/interactivity/src/*`

Reply:
Makes sense. I guess we can use the `packages/interactivity*/src/**` existing group?

Reply:
Done in 54f002a95c8a06b9449e44f30f79f8855b731d3e